.github/workflows/packages.yaml

```diff
@@ -10,8 +10,40 @@ on:
   workflow_dispatch:
 
 jobs:
-  build-distro-packages:
-    name: build packages
+  build-distro-packages-arm64:
+    runs-on: ubuntu-latest
+    name: build arm64 packages
+    strategy:
+      fail-fast: true
+      matrix:
+        format: [ rpm, deb ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: uraimo/run-on-arch-action@v2.1.1
+        name: Build the ${{matrix.format}} packages
+        with:
+          arch: aarch64
```

Comment:
Could we not just use this action for them all and add the arch as part of the matrix?

Reply:
Unfortunately, the action is very slow since it does cross-compile via qemu. I'd prefer to keep the native amd64 target as is now